### DISCUSSION TITLE
Is it viable to use the CVE-2023-2033 exploit?

### DISCUSSION THREAD

**Discussion by py660** [3kh0/ext-remover#610] [Polls JOIN TITANIUM NETWORK TO SHARE PLEASE]
I've recently started to see news article saying that the "type confusion" exploit could allow hackers to run malicious (or in this case, extension-removing) code. Better yet, the chrome update will take anywhere from a couple of days to a few weeks to push, and when paired with school's lack of motivation to force a timely upgrade, this type of exploit *could* work on chromebooks not on v112.

**Comment by funkymokey** [comment]
my chromebook dosent even update to 109 because its unsupported

**Comment by 3kh0** [comment]
CVE research links:
NVD: https://nvd.nist.gov/vuln/detail/CVE-2023-2033
CVE: https://www.cve.org/CVERecord?id=CVE-2023-2033

**Reply by TheAlterning** [reply]
do you plan to make anothe ext remover with this?

**Reply by amelia-bartlett** [reply]
Debian bugtracker (same issue): https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=1034406

**Comment by py660** [comment]
It *could* be possible.

On Tue, Apr 18, 2023 at 2:41 PM TheAlterning ***@***.***>
wrote:

> do you plan to make anothe ext remover with this?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/3kh0/ext-remover/discussions/610#discussioncomment-5652684>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AWKSL6SUXOVSKYL357OZQBTXB3N6BANCNFSM6AAAAAAXB5OY34>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
[image: ***@***.*** ***@***.***>
